### DISCUSSION TITLE
Contributions from external contributions

### DISCUSSION THREAD

**Discussion by DVGY** [mondaycom/vibe#1502] [Q&A] [answered]
Is it possible to start contributions, I am not from Monday Team. Does Monday allow external contributions ?

Thanks

**Comment by shlomitc** [comment, marked as answer]
Hey @DVGY 
Thanks for raising this question.
As of today, we're working mainly as a core team with some contributions from monday.com employees, however, we're always open to contributions according to our guidelines.

With that being said, not every contribution will be accepted as we try to keep our design and APIs according to the monday.com design language.

Do you have any specific contribution you would like to make?
